packages/components/src/tooltip/index.tsx

```diff
@@ -64,17 +87,30 @@ function Tooltip( props: TooltipProps ) {
 	}
 	computedPlacement = computedPlacement || 'bottom';
 
+	// Disable reason: the hook can not run conditionally.
+	// eslint-disable-next-line @wordpress/no-unused-vars-before-return
```

Comment:
Adding the early return when `isNestedInParentTooltip` causes ESlint to complain that `tooltipStore` may be unused before `return`. But hooks can't be run conditionally, and therefore I disabled the ESlint warning.